source/renderer/app/utils/formatters.js

```diff
@@ -4,12 +4,55 @@ import {
   DECIMAL_PLACES_IN_ADA,
   LOVELACES_PER_ADA,
 } from '../config/numbersConfig';
+import { NUMBER_FORMATS } from '../../../common/types/number.types';
 
+// Symbol	  Name	              Scientific Notation
+// K	      Thousand	          1.00E+03
+// M	      Million	            1.00E+06
+// B	      Billion	            1.00E+09
+// T	      Trillion	          1.00E+12
+// Q	      Quadrillion	        1.00E+15
 export const formattedWalletAmount = (
```

Comment:
@DeeJayElly please improve this logic because app crashes.
How to reproduce:
1. Create new wallet (empty)
2. Send 1ADA to newly created wallet and you will see formattedAmount split() function failure
<img width="1141" alt="Screen Shot 2019-11-25 at 12 31 19" src="https://user-images.githubusercontent.com/18653950/69537226-f857fa80-0f7f-11ea-90cc-ae02cafa64d3.png">
![Screen Shot 2019-11-25 at 12 30 49](https://user-images.githubusercontent.com/18653950/69537227-f857fa80-0f7f-11ea-861d-1d2bfd9de413.png)